package.json

```diff
@@ -18,5 +18,8 @@
   "homepage": "https://github.com/projectcypress/js-ecqm-engine#readme",
   "devDependencies": {
     "jasmine": "^2.9.0"
+  },
+  "dependencies": {
+    "mongoose": "^5.0.3"
```

Comment:
Can you commit the resultant `package-lock.json` from running `npm install` here? It should be commited per https://docs.npmjs.com/files/package-lock.json

Reply:
@okeefm Just a thought, have we considered using Yarn for js dependency management? Not a biggie, but I'm not sure we ever talked about dependency management.